.github/dependabot.yml

```diff
@@ -1,19 +1,12 @@
 version: 2
```

Comment:
Just FYI:
- Currently security updates only are enabled.
- By adding this file, you will switch to ALL updates, not just security ones.

Reply:
@aravindvnair99 was this your intention?